installation/index.md

```diff
@@ -36,13 +36,16 @@ The previously mentioned [openHABian](openhabian.html) can also be used to kicks
 
 ## Prerequisites
 
-Make sure that you have an up to date version of the [Oracle Java Runtime Environment](https://java.com/en/download/manual.jsp) on your host system installed.
-A famous choice is OpenJDK, however, Oracle Java Version 8 is recommended for openHAB.
-openHAB may be unstable and slower executed by OpenJDK.
+Make sure that you have an up to date Java Runtime Environment (JRE) installed on your host system. The [Oracle JRE](https://java.com/en/download/manual.jsp)
+is recommended for openHAB. Alternative JRE implementations such as OpenJDK may be used, but there are some known limitations with other JREs depending on the platform and bindings used. Your milage may vary.
 
-> Oracle Java offers significant performance advantages over OpenJDK on ARM platforms, and will expand the range of Java applications that run well [on the Raspberry Pi].
+| JRE | Advantages | Disadvantages |
+| --- | --- | --- |
+| Oracle | Full openHAB support on all platforms | Not supported by some Linux distributions (packages may not be available, requiring manual installation) |
+| OpenJDK | Supported by most Linux distributions (packages typically available) | Performance issues on ARM platforms, compatibiltiy issues with certain bindings and certificates | 
+| Zulu Embedded | Optimized for ARM platforms, fully compliant with Java SE standard | Not available on all operating systems |
```

Comment:
Hey @stephenbonar,
great you reacted on the discussion we had on the forum ;) I like the table in general but please add a few links to the mix. Every name should be linked to it's official website and things like "Performance issues" need proof, as discussed in the forum.

I'm also not sure if you should talk about "JRE" as many will be confused by the JRE vs. JDK question. I'm not sure if "Java platform" or "JVM" is the better term to use.